cmd/resource-topology-exporter/main.go

```diff
@@ -25,13 +24,29 @@ const (
 	ProgramName = "resource-topology-exporter"
 )
 
+type ProgArgs struct {
+	NRTupdater      nrtupdater.Args
+	Resourcemonitor resourcemonitor.Args
+	RTE             resourcetopologyexporter.Args
+	Version         bool
+}
+
+func (pa *ProgArgs) ToJson() ([]byte, error) {
+	return json.Marshal(pa)
+}
+
 func main() {
-	nrtupdaterArgs, resourcemonitorArgs, rteArgs, err := argsParse(os.Args[1:])
+	parsedArgs, err := parseArgs(os.Args[1:]...)
 	if err != nil {
-		log.Fatalf("failed to parse command line: %v", err)
+		log.Fatalf("failed to parse args: %v", err)
 	}
 
-	cli, err := podrescli.NewFilteringClient(rteArgs.PodResourcesSocketPath, rteArgs.Debug, rteArgs.ReferenceContainer)
+	if parsedArgs.Version {
+		fmt.Println(ProgramName, "TODO add Version track logic")
```

Comment:
do we have plans to address this?

Reply:
Yes, we do. would you like me to issue a card?

Reply:
If  you can file a PR today, no need for a card; otherwise yes, please do.

Reply:
Will file a PR today